packages/analytics/analytics-chart/src/components/AnalyticsChart.vue

```diff
@@ -43,10 +47,17 @@
         class="dropdown"
         data-testid="chart-action-menu"
       >
-        <MoreIcon
-          :color="KUI_COLOR_TEXT_NEUTRAL"
-          :size="KUI_ICON_SIZE_40"
-        />
+        <button
+          appearance="none"
+          :aria-label="i18n.t('more_actions')"
+          class="kebab-action-menu"
+          data-testid="kebab-action-menu"
+        >
```

Comment:
good call here, without the button element, the nested SVG can sometimes "steal" the click action